sceptre/template.py

```diff
@@ -215,7 +215,8 @@ def get_boto_call_parameter(self):
         :returns: The boto call parameter for the template.
         :rtype: dict
         """
-        if self.s3_details:
+        # If bucket_name is set to None, it should be ignored and not uploaded.
+        if self.s3_details and self.s3_details.get("bucket_name") is not None:
```

Comment:
I believe the `is not None` can be shortend to simply `if self.s3_details and self.s3_details.get("bucket_name")` here?